train/data_parser.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import os
+import numpy as np
+import tensorflow as tf
+import struct
+import functools
+from tictactoe import TicTacToe
+
+
+class DataParser:
+    def __init__(self, game, data_dir):
+        self.game = game
+        self.data_dir = data_dir
+
+    def _data_entries_filenames_gen(self):
+        filenames = os.listdir(self.data_dir)
+        filenames = [os.path.join(self.data_dir, filename)
+                     for filename in filenames]
+
+        # take the latests files_count files
+        files_count = 100000

```

Comment:
Why? should this be a parameter?

Reply:
for sure, didn't implement it all the way. issue opened